packages/waku/src/lib/renderers/html-renderer.ts

```diff
@@ -102,6 +103,13 @@ globalThis.__WAKU_PREFETCHED__ = {
       if (scriptsClosed) {
         return;
       }
+      // enqueue `mainJs` first in the body to avoid the document.body undefined error, before we used to inject it in the head which sometimes made it load before the document.body was loaded
+      controller.enqueue(
+        encoder.encode(
+          `<script src="${config.basePath}${config.srcDir}/${config.mainJs}" async type="module"></script>`,
+        ),
+      );
+
```

Comment:
Here, this adds `<script ...>` multiple times.
`npm run examples:dev:04_promise` can reproduce it.